beacon-chain/p2p/peers/status.go

```diff
@@ -475,7 +502,33 @@ func (p *Status) OutboundConnected() []peer.ID {
 	return peers
 }
 
-// Active returns the peers that are connecting or connected.
+// OutboundConnected returns the current batch of outbound peers that are connected using TCP.
+func (p *Status) OutboundConnectedTCP() []peer.ID {
```

Comment:
Same here

Reply:
Fixed in https://github.com/prysmaticlabs/prysm/pull/13786/commits/f42455b6acf2323e38729bd3979cf60cd283ea0f.